src/main/java/edu/cornell/kfs/sys/CUKFSConstants.java

```diff
@@ -22,6 +22,7 @@
     public static final String DATE_FORMAT_mm_dd_yyyy_hh_mm_ss_am = "MM/dd/yyyy hh:mm:ss a";
     public static final String DATE_FORMAT_yyyy_MM_dd_T_HH_mm_ss_SSS = "yyyy-MM-dd'T'HH:mm:ss.SSS";
     public static final String DATE_FORMAT_dd_MMM_yy = "dd-MMM-yy";
+    public static final String DATE_FORMAT_YYYY_HYPHEN_MM_HYPHEN_DD = "yyyy-MM-dd";
```

Comment:
Should the HYPHEN word just be removed from this constant name?
Also, you may be able to used this existing constant CUKFSConstants.DATE_FORMAT_yyyy_MM_dd instead.

Reply:
good catch, I shall switch